packages/components/visual-hidden/src/visual-hidden.vue

```diff
@@ -4,36 +4,31 @@
   </span>
 </template>
 
-<script lang="ts">
-import { computed, defineComponent } from 'vue'
+<script lang="ts" setup>
+import { computed } from 'vue'
+import { visualHiddenProps } from './visual-hidden'
+import type { StyleValue } from 'vue'
+const props = defineProps(visualHiddenProps)
 
-export default defineComponent({
+defineOptions({
   name: 'ElVisuallyHidden',
-  props: {
-    style: {
-      type: [String, Object, Array],
+})
+
+const computedStyle = computed(() => {
```

Comment:
It may be easier to understand by changing the way of writing here.

```diff
- const computedStyle = computed(() => {
+ const computedStyle = computed<StyleValue>(() => {
```